### PR TITLE
New zombie type: Sleepwalker

### PR DESCRIPTION
**Pull request by Daved27hundred**
#### Summary
Content "Added the Sleepwalker zombie, a zombie filled with relaxation gas that bursts like a bloated zombie."

#### Purpose of change

The sleepwalker uses the same relaxation gas that riot control bots use. When inhaled, it will prevent survivors from attacking. This makes the sleepwalker a deadly addition to a horde and will make survivors will think twice about engaging it.

#### Describe alternatives you've considered

not adding it

#### Testing

currently in progress

### CODE REVIEW

File: data/json/monsters/zed_explosive.json [hunk 1]
```diff
@@ -176,6 +176,38 @@
     "fungalize_into": "mon_zombie_gasbag_fungus",
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
   },
+  {
+    "id": "mon_zombie_relax_gasbag",
+    "type": "MONSTER",
+    "name": { "str": "sleepwalker zombie" },
+    "description": "A zombie swollen with a blueish, oddly sweet smelling gas. It stumbles around even more than regular zombies, as if it were heavily intoxicated.",
```
Comment (NetSysFire) [inline data/json/monsters/zed_explosive.json:183]:
```suggestion
    "description": "A zombie swollen with a bluish, oddly sweet smelling gas. It stumbles around even more than regular zombies, as if it was heavily intoxicated.",
```

Comment (J-Fields) [reply data/json/monsters/zed_explosive.json:183]: Maybe even just "as if heavily intoxicated"?

Comment (Daved27hundred) [reply data/json/monsters/zed_explosive.json]: done.

File: data/json/monsters/zed_explosive.json [hunk 2]
```diff
@@ -176,6 +176,38 @@
...
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 0,
+    "armor_elec": 1,
+    "vision_night": 3,
+    "harvest": "exempt",
+    "emit_fields": [ { "emit_id": "emit_relax_gas_leak", "delay": "1 s" } ],
+    "special_attacks": [ [ "SUICIDE", 20 ], [ "scratch", 15 ] ],
+    "death_drops": "default_zombie_items",
+    "death_function": { "message": "A %s explodes!", "effect": { "id": "death_gas", "hit_self": true }, "corpse_type": "NO_CORSPE" },
```
Comment (J-Fields) [inline data/json/monsters/zed_explosive.json:207]:
```suggestion
    "death_function": { "message": "A %s explodes!", "effect": { "id": "death_gas", "hit_self": true }, "corpse_type": "NO_CORPSE" },
```

Comment (Daved27hundred) [reply data/json/monsters/zed_explosive.json:207]: oh god why did that slip by me

Comment (GuardianDll) [reply data/json/monsters/zed_explosive.json:207]: you can press "commit suggestion" and it change it, so you dont need to fix it manually

File: data/json/monsters/zed_explosive.json (hunk 1 above)
Comment (J-Fields) [reply data/json/monsters/zed_explosive.json]: I think you missed the "bluish" change. Seems like "blueish" is an accepted spelling, but "bluish" is definitely more common.

Comment (NetSysFire) [reply data/json/monsters/zed_explosive.json]: True. But we have blueish nowhere currently but "bluish" gives me two hits with grep -r. Nonetheless a very small issue.

**Comment by Daved27hundred** [comment]
Apparently magenta_blue isn't a valid color. if anyone wants to change that be my guest, I have a dentist appointment.

**Comment by Daved27hundred** [comment]
doing what someone suggested on discord and changing magenta_blue to magenta_cyan

**Pull request merged**